matRadGUI.m

```diff
@@ -1044,7 +1044,7 @@ function UpdatePlot(handles)
     end
 end
 
-if get(handles.radioBtnIsoCenter,'Value') == 1 && get(handles.popupTypeOfPlot,'Value') == 1 && ~isempty(pln)
+if get(handles.radioBtnIsoCenter,'Value') == 1 && get(handles.popupTypeOfPlot,'Value') == 1 && ~isempty(pln) && size(pln.isoCenter,1) == 1
     hIsoCenterCross = matRad_plotIsoCenterMarker(handles.axesFig,pln,ct,plane,slice);
```

Comment:
we can also plot if all are the same! check with unique command and option rows.